spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -229,6 +230,20 @@ public String stop(@CliOption(key = { "", "ids" }, help = "the task execution id
 		return String.format("Request to stop the task execution with id(s): %s has been submitted", ids);
 	}
 
+	@CliCommand(value = LOG, help = "Retrieve task execution log")
+	public String retrieveTaskExecutionLog(@CliOption(key = { "", "id" }, help = "the task execution id", mandatory = true) long id,
+			@CliOption(key = { "", "platform" }, help = "the platform of the task execution", mandatory = false) String platform) {
```

Comment:
We can remove key `""` option for the platform key as we can only have one of the options with the default empty key.